clarity/contracts/arkadiko-auction-engine-v1-1.clar

```diff
@@ -63,6 +63,7 @@
 (define-data-var auction-ids (list 1500 uint) (list u0))
 (define-data-var lot-size uint u1000000000) ;; 1000 xUSD
 (define-data-var auction-engine-shutdown-activated bool false)
+(define-data-var removing-auction-id uint u0)
```

Comment:
Q: why do we need to store this as a var on the SC?

Reply:
else we don't know which auction we're removing in the `filter` method on line 604